src/controller/kafka/kafka_controller.py

```
@@ -15,12 +15,13 @@
 @version(1)
 @with_kafka_broker_connection
 async def send_test_message_to_kafka(
+    topic: str,
     current_user: UserRead = Depends(get_current_user),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)